list.go

```diff
@@ -59,16 +61,25 @@ func init() {
 	listTypeNameMapper.Store("float64", "[double")
 	listTypeNameMapper.Store("bool", "[boolean")
 	listTypeNameMapper.Store("time.Time", "[date")
+
+	listTypeNameMapper.Store("hessian.Object", "[object")
 }
 
 func registerTypeName(gotype, javatype string) {
 	listTypeNameMapper.Store(gotype, "["+javatype)
 }
 
 func getListTypeName(gotype string) string {
-	v, ok := listTypeNameMapper.Load(gotype)
+	buf := strings.Builder{}
+	count := strings.Count(gotype, "[]")
+	for i := 0; i < count; i++ {
+		buf.WriteString("[")
+	}
+	gotype = strings.Replace(gotype, "[]", "", -1)
+	v, ok := listTypeNameMapper.Load(strings.TrimPrefix(gotype, "*"))
```

Comment:
the value of listTypeNameMapper already contains `[` prefix, the result of `getListTypeName` will contains one more `[`.

Reply:
`listTypeNameMapper` need to contains `[` prefix. What is the problem.

Reply:
if the count=1, `for i := 0; i < count; i++ { buf.WriteString("[") }` write a prefix `[`,  and  `buf.WriteString(v.(string))` write `[object` again, the result will be `[[object`

Reply:
list.go#Line154 : `value.Type().Elem()` will get a element instead of array. So it need add a `[`.